stdlib/sql/to.go

```diff
@@ -263,7 +263,7 @@ func getTranslationFunc(driverName string) (func() translationFunc, error) {
 	switch driverName {
 	case "sqlite3":
 		return SqliteColumnTranslateFunc, nil
-	case "postgres", "sqlmock":
+	case "postgres", "sqlmock", "vertica", "vertigo":
```

Comment:
Is `vertigo` another name for Vertica databases?

Reply:
An issue from our side is that the driver packages were renamed from "vertigo" to "vertica" at some point, so this is done as a guard for users who might have pulled or cached older GO code.  So yes, it's a deprecated alias.